tests/consensus_tests/test_resharding.py

```diff
@@ -2,991 +2,307 @@
-            })
-        assert_http_ok(r)
-        wait_for_collection_shard_transfers_count(peer_api_uris[0], COLLECTION_NAME, 0)
-        assert check_collection_local_shards_count(peer_api_uris[0], COLLECTION_NAME, 1)
-
-    # Assert node point count
-    for uri in peer_api_uris:
-        assert get_collection_point_count(uri, COLLECTION_NAME, exact=True) == num_points
-
-    # Reshard 5 times in sequence
-    for _shard_count in range(5):
-        # Start resharding
-        r = requests.post(
-            f"{peer_api_uris[0]}/collections/{COLLECTION_NAME}/cluster", json={
-                "start_resharding": {
-                    "direction": "up",
-                    "peer_id": first_peer_id
-                }
-            })
-        assert_http_ok(r)
-
-        # Wait for resharding operation to start and stop
-        wait_for_collection_resharding_operations_count(peer_api_uris[0], COLLECTION_NAME, 1)
-        for uri in peer_api_uris:
-            wait_for_collection_resharding_operations_count(uri, COLLECTION_NAME, 0)
-
-        # Point count across cluster must be stable
-        for uri in peer_api_uris:
-            assert get_collection_point_count(uri, COLLECTION_NAME, exact=False) == num_points
-
-    # We must end up with:
-    # - 6 shards on first node, it was the resharding target
-    # - 3 shards on the other two nodes, 6 replicas balanced over 2 nodes
-    assert check_collection_local_shards_count(peer_api_uris[0], COLLECTION_NAME, 6)
-    for uri in peer_api_uris[1:]:
-        assert check_collection_local_shards_count(uri, COLLECTION_NAME, 3)
-
-
-# Test resharding with concurrent updates.
-#
-# This performs resharding a few times while sending point updates to all peers
-# concurrently. At the end of the whole process it asserts the expected point
-# count.
-#
-# The concurrent update tasks consist of:
-# - 3 threads upserting new points on all peers
-# - 1 threads updating existing points on the first peer
-# - 2 threads deleting points on the first two peers
-@pytest.mark.skip(reason="moving resharding driver to external service")
-def test_resharding_concurrent_updates(tmp_path: pathlib.Path):
-    assert_project_root()
+    # Wait for resharding to abort
+    wait_for_collection_resharding_operations_count(peer_uris[0], COLLECTION_NAME, 0)
 
-    num_points = 1000
-    num_inserts = 1000
-    num_updates = 500
-    num_deletes = 33
+def test_resharding_abort_on_delete_collection(tmp_path: pathlib.Path):
+    # Bootstrap resharding cluster
+    peer_uris, peer_ids = bootstrap_resharding(tmp_path, peer_idx=-1)
 
-    # Prevent optimizers messing with point counts
-    env={
-        "QDRANT__STORAGE__OPTIMIZERS__INDEXING_THRESHOLD_KB": "0",
-    }
+    # Remove target peer
+    resp = requests.delete(f"{peer_uris[0]}/collections/{COLLECTION_NAME}")
+    assert_http_ok(resp)
 
-    peer_api_uris, _peer_dirs, _bootstrap_uri = start_cluster(tmp_path, 3, None, extra_env=env)
+    # TODO: Wait for/check... *what*? 🤔
```

Comment:
We cannot check this.

Maybe we want to assert the collection is properly deleted, I don't think we can do more.